Sources/third_party/Experimental/Complex.swift

```diff
@@ -0,0 +1,325 @@
+// A major part of the Complex number implementation comes from
+// [@xwu](https://github.com/xwu)'s project, [NumericAnnex](https://github.com/xwu/NumericAnnex).
+
+struct Complex<T : FloatingPoint> {
```

Comment:
In the swift-apis repository, we adopt the [Google Swift Style Guide](https://google.github.io/swift/) with the exception that indentation is 4 spaces long. One things that is different from the style we use in the standard library is that we leave no space before colons in conformance constraints. That is, `T : P` should be changed to `T: P`.
```suggestion
struct Complex<T: FloatingPoint> {
```
Same for other occurrences of conformance constraints in these files.

Reply:
Will make sure to reference that next time I'm submitting a PR in one of our repos! I will have to update my [Protocol Oriented](https://github.com/tensorflow/swift/pull/187) walkthrough as well.